src/structures/Node.ts

```diff
@@ -377,38 +363,33 @@ export class Node {
     }
 
     /**
-     * Update Player Data
-     * @param player
-     * @param payload
+     * Update the player's data.
+     * 
+     * @param player The player.
+     * @param payload The payload data to be sent while updating.
      * @protected
      */
     protected updatePlayerData(player: Player, payload: Payload) {
-        player.paused = payload.d.paused;
+        player.playing = !payload.d.paused;
```

Comment:
I think keeping paused is better since playing can be false for 2 reason once when it is paused and another when it is stopped. It will help in differentiating

Reply:
You are right but see `paused = !playing`. Its not always like that, in the situate where the state of the player is `connecting` or `disconnected` the playing will be `false` and that means paused will be `true` so i already made a getter to do that stuff. Kindly the check the new code which i commited already (https://github.com/Scientific-Guy/chiro.js/blob/master/src/structures/Manager.ts)

Reply:
Well keeping the `playing` as property and `paused` as getter is good.